models.py

```diff
@@ -72,19 +71,20 @@ class Employee(Base):
         server_default=text("nextval('person_person_id_seq'::regclass)"))
     manager_person_id = Column(ForeignKey('person.person_id'))
     account_manager_flag = Column(Boolean)
-    current_employee_flag = Column(Boolean)
-
+    current_employee_flag = Column(Boolean, server_default=text("true"))
```

Comment:
Does text("true") make it boolean true? Just a bit confused with a Boolean type and default text value.